.gitattributes

```diff
@@ -0,0 +1,4 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
```

Comment:
change path to external here to avoid potential regressions.

Reply:
I don't need this file anymore :)